docs/docs/overview/quickstart/get-started-with-python.md

```diff
@@ -0,0 +1,278 @@
+| ------- | --------------------------------------------------------------------------------------------- |
+| Email   | 1. Custom Code (HTML) with option to use custom variables via the handlebars , {{ }}, syntax. |
+|         | 2. Click and place UI items with the visual template editor.                                  |
+| SMS     | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| Chat    | Text with the option to use handlebars syntax, {{ }} to inject custom variables.              |
+| In-App  | Text                                                                                          |
+
+:::info
+Proper authorization needs to be set for the Chat channel for subscribers.
+:::
+
+To create a notification workflow, please follow the following steps:
+
+1. Click “Workflows” on the left sidebar of your Novu dashboard.
+2. Click the “Create Workflow” button on the top right.
+3. Select 'blank workflow' from the dropdown.
+4. The name of a new workflow is currently "Untitled." Rename it to a suitable title.
+5. Select any channel you want to use in your app. For the sake of this guide, we'll be using the 'Email' channel.
+   ![set-email.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/set-email_wavtrn.png)
+
+6. Click on the recently added channel, fill the email subject and click “Update”.
+   ![update_email_template.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/update_email_template_ivn0jv.png)
+
+7. Click on the “Test” tab and send a test email to verify your notification workflow.
+   ![send_test_email.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1686776583/send_test_email_ngzmth.png)
+
+You should get an email within seconds. If you didn't, please check your 'spam' folder as sometimes test emails can end up there. Yaaay, you have successfully sent your first notification via the Novu dashboard!
+
+Now, let’s take it a step further to trigger notifications via code.
+
+## Create A Subscriber
+
+The recipients of a triggered notification are called subscribers.
+
+Click “Subscribers” on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display only one subscriber, as you were added automatically during sign-up.
+
+![subscriber_id.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1688331839/Screenshot_2023-07-03_at_0.02.53_jmkhi3.png)
+
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+
+```python
+# Create a subscriber
+
+from novu.api.subscriber import SubscriberApi
+from novu.dto.subscriber import SubscriberDto
+   
+your_subscriber_id = "123" # Replace this with a unique user ID.
+
+# Define a subscriber instance
+subscriber = SubscriberDto(
+   subscriber_id=your_subscriber_id,
+   email="abc@gmail.com",
+   first_name="John",
+   last_name="Doe"
+)
+
+SubscriberApi().create(subscriber)
```

Comment:
I am not sure, it will take apiUrl from config or we need to put API URL like this
https://github.com/novuhq/novu-python#subscribers

Reply:
It will. There are two ways to do it.

Set the apiUrl from config so it uses throughout all the API calls or put in the API url for each api method you call. I chose the former @jainpawan21

Reply:
Okay, Rest looks fine to me.